[2022-12-04 19:20:22,741] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: testing_aws_wrangler.read_data_with_awswrangler manual__2022-12-04T19:20:17.779546+00:00 [queued]>
[2022-12-04 19:20:22,747] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: testing_aws_wrangler.read_data_with_awswrangler manual__2022-12-04T19:20:17.779546+00:00 [queued]>
[2022-12-04 19:20:22,747] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 19:20:22,747] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-04 19:20:22,747] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 19:20:22,757] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): read_data_with_awswrangler> on 2022-12-04 19:20:17.779546+00:00
[2022-12-04 19:20:22,762] {standard_task_runner.py:52} INFO - Started process 570 to run task
[2022-12-04 19:20:22,764] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'testing_aws_wrangler', 'read_data_with_awswrangler', 'manual__2022-12-04T19:20:17.779546+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/test_pandas.py', '--cfg-path', '/tmp/tmpssqlvgmf', '--error-file', '/tmp/tmpunvr3x77']
[2022-12-04 19:20:22,765] {standard_task_runner.py:80} INFO - Job 12: Subtask read_data_with_awswrangler
[2022-12-04 19:20:22,802] {task_command.py:369} INFO - Running <TaskInstance: testing_aws_wrangler.read_data_with_awswrangler manual__2022-12-04T19:20:17.779546+00:00 [running]> on host 096df828fe38
[2022-12-04 19:20:22,854] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=nic@enye.tech
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=testing_aws_wrangler
AIRFLOW_CTX_TASK_ID=read_data_with_awswrangler
AIRFLOW_CTX_EXECUTION_DATE=2022-12-04T19:20:17.779546+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-04T19:20:17.779546+00:00
[2022-12-04 19:20:22,867] {credentials.py:1120} INFO - Found credentials in environment variables.
[2022-12-04 19:20:23,160] {python.py:173} INFO - Done. Returned value was: (76518, 20)
[2022-12-04 19:20:23,181] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=testing_aws_wrangler, task_id=read_data_with_awswrangler, execution_date=20221204T192017, start_date=20221204T192022, end_date=20221204T192023
[2022-12-04 19:20:23,216] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-04 19:20:23,241] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
